pkg/tenant/tenants_test.go

```diff
@@ -11,6 +11,31 @@ import (
 	"github.com/trisacrypto/directory/pkg/trtl/pb/v1"
 )
 
+func (suite *tenantTestSuite) TestTenantList() {
```

Comment:
I get the following message when trying to run the test: `runtime error: invalid memory address or nil pointer dereference`

The stack trace points to line 20 of the tenants.go file. I would think it's because I've used a `_`, but it doesn't seem like I can use a variable declared there. Although, I could be missing something.

Reply:
See comment above on line 20